dolphin/task_manager/manager.py

```diff
@@ -63,3 +64,44 @@ def say_hello(self, context, request_spec=None,
 
         except Exception as ex:
             pass
+
+    def get_volumes(self, context, request_spec=None, filter_properties=None):
+        # 1. Call the list volumes
```

Comment:
1. sync_volume or refresh_volume is better
2. why need request_spec or filter_properties?

Reply:
i will remove request_spec or filter_properties. I have kept, becasue we may need some parameters during driver call.

sync_volumes has some different meanings. actually we are retrieving all volumes, so get_volumes would be better name.